ssas/connection.go

```diff
@@ -0,0 +1,48 @@
+package ssas
+
+import (
+	"database/sql"
+	"log"
+	"os"
+	"runtime"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	_ "github.com/lib/pq"
+)
+
+// Variable substitution to support testing.
+var LogFatal = log.Fatal
+
+func GetDbConnection() *sql.DB {
+	databaseURL := os.Getenv("DATABASE_URL")
```

Comment:
Is it too soon to specify a `ssas`-specific database URL?

Reply:
That one is more complicated to change, because it is used in many different places: launch scripts, makefiles, etc. I was saving it for later when we're done with writing new code and are working on integration.